syntax="proto3";

package greet.v1;

service Greeter {
    rpc SayHello (HelloRequest) returns (HelloReply) {}
    rpc SayHelloSecure (HelloRequest) returns (HelloReply) {}
    
}

message HelloRequest {
    string name = 1;
}
  
message HelloReply {
    string message = 1;
}
